tests-system/rbt-signature-matches-3/foo.rsl

```diff
@@ -0,0 +1,11 @@
+package Foo
+
+type T {
+    a String
+    b String
+    c Integer
+}
+
+checks T {
+    matches(a, "(abc(def)"), warning "a should match with potato", a
```

Comment:
The hint is wrong. This does not matter for the test, but it might confuse ourselves in the future.
Please update as follows: `"This check uses an invalid regex, and the test must show that the parser rejects it"`
Or like this: `"a should match with the regex (abc(def))"`